cli/azd/cmd/monitor.go

```diff
@@ -164,19 +158,19 @@ func (m *monitorAction) Run(ctx context.Context) (*actions.ActionResult, error)
 	for _, insightsResource := range insightsResources {
 		if m.flags.monitorLive {
 			openWithDefaultBrowser(
-				fmt.Sprintf("https://app.azure.com/%s%s/quickPulse", account.TenantId, insightsResource.Id),
+				fmt.Sprintf("https://app.azure.com/%s/quickPulse", insightsResource.Id),
```

Comment:
`tenantId` isn't required. It only works when the user isn't logged in, that it would set the correct tenant login flow. If the user is already logged in, and has a directory set to a different tenant, it simply fails.

I might `tenantId` back now that I've figured out all the other peieces.

Reply:
Restored `tenantId` formatting, which is now picked correctly based on the account's subscription access.